Add audacious-1.3 support to g15daemon_audacious_spectrum-1.0pre

Via
Alex Maclean <monkeh@monkeh.net>

--- g15daemon_audacious_spectrum.c.orig	2007-03-11 18:48:14.000000000 +0000
+++ g15daemon_audacious_spectrum.c	2007-03-11 18:50:15.000000000 +0000
@@ -195,15 +195,16 @@
     TitleInput *tuple;
     char *strtok_ptr;
     static int vol_timeout=0;
+	Playlist *current = playlist_get_active();
 
         pthread_mutex_lock (&g15buf_mutex);
 	g15r_clearScreen (canvas, G15_COLOR_WHITE);
 
-	if (playlist_get_length() > 0)
+	if (playlist_get_length(current) > 0)
 	  {
-	        playlist_pos = playlist_get_position();
+	        playlist_pos = playlist_get_position(current);
 		
-	        tuple = playlist_get_tuple(playlist_pos);
+	        tuple = playlist_get_tuple(current,playlist_pos);
 		title = xmms_get_titlestring("%p - %t", tuple);
 	        if(title!=NULL){ //amarok doesnt support providing title info via xmms interface :(
 	            if(strlen(title)>32) {
